cap/modules/records/jsonschemas/definitions/MonteCarloDataset-vtest01.json

```diff
@@ -32,7 +32,7 @@
       "description": "",
       "type": "array",
       "items": {
-        "$ref": "/static/jsonschemas/definitions/Identifiers-vtest01.json#"
+        "$ref": "http://localhost:5000/records/jsonschemas/definitions/Identifiers-vtest01.json"
```

Comment:
a) keep relative path
b) add analysis-preservation.cern.ch + add rule for jsonresolver